cosinnus/static/less/v2/dashboard.less

```diff
@@ -484,25 +485,19 @@
 			.comment-form {
 				cursor: text;
 				padding: 0;
-				margin: 0 37px -20px 37px;
+				margin: 0 37px 0px 37px;
 				
 				textarea {
 					border: none;
 					width: 100%;
-					padding-left: 0;
+					padding-left: 4px;
 					padding-right: 0;
 					line-height: 18px;
 					height: 38px;
-					&:focus {
```

Comment:
Das Verstecken der Outline für fokussierte Elemente is a11y bad practice. Daher auch etwas `padding-left` hinzugefügt, damit die Outline nicht direkt am ersten Buchstaben klebt